### DISCUSSION TITLE
Upcoming releases

### DISCUSSION THREAD

**Discussion by erez-tom** [google-deepmind/mujoco#126] [Announcements]
It’s been just over a month since the release of MuJoCo 2.1.1, and we would like to give you an update on our planned release schedule. We intend to have one last binary-only release in late February, which will focus on new Python bindings. The following release, currently planned for April, will be the open source release.

**Comment by bhartsb** [comment]
Great, I'm really looking forward to the open source release.

**Comment by kevinzakka** [comment]
Thanks for the update @erez-tom !

Out of curiosity, will this break current functionality built on top of 2.1.1?

**Comment by MyelinsheathXD** [comment]
Yeah ! The open source one is the one that  we have been waiting for!👍

**Comment by yuvaltassa** [comment]
The python bindings are taking a bit longer than we expected, so our upcoming binary release (2.1.2) is now slated for the first half of March. This may impact our planned timeframe for open-sourcing. In any case we will keep posting updates on this thread.

**Comment by doodlesbykumbi** [comment]
Extremely excited for the open source release ✨

**Comment by doodlesbykumbi** [comment]
Hi. Just checking in to see how that open source release is looking? Very much excited!

**Comment by bhartsb** [comment]
Ditto that.

**Comment by yuvaltassa** [comment]
Hi folks, thanks for checking in.
There have been some minor delays, we are currently targeting mid May as the most likely timeframe for open sourcing.
Cheers,

**Comment by huyz-git** [comment]
Any update on this?

**Comment by bhartsb** [comment]
Yes I'm interested to find out as well.

**Comment by saran-t** [comment]
This came out [today](https://www.deepmind.com/blog/open-sourcing-mujoco).

**Comment by bhartsb** [comment]
This is great news.  Thank you.
